seatunnel-common/src/main/java/io/github/interestinglab/waterdrop/common/RowConstant.java

```diff
@@ -14,10 +14,11 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.github.interestinglab.waterdrop.common
 
-object RowConstant {
-  val ROOT = "__root__"
-  val TMP = "__tmp__"
-  val JSON = "__json__"
-}
\ No newline at end of file
+package io.github.interestinglab.waterdrop.common;
+
+public class RowConstant {
```

Comment:
how about change class to interface type? or you should add private constructor.